examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb

```diff
@@ -25,7 +25,7 @@
     "\n",
```

Comment:
<p>This blurb is probably sufficient for all the other examples after the getting started, as opposed to the longer explanation of hybrid jobs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Are there no other examples of creating jobs from scripts that we can link from?</p>